src/components/Category/AddCategoryComponent.jsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Label } from 'semantic-ui-react';
+import ArtButton from '../common/ButtonComponent';
+import InputFluid from '../common/TextInputComponent';
+
+import '../../_css/AddAssetComponent.css';
+
+const AddCategoryComponent = props => (
+  <div>
```

Comment:
The outer `div` is not needed. `<Form>` is already a parent component

Reply:
done.